src/k8s/cmd/k8s/k8s_local_node_status.go

```diff
@@ -0,0 +1,29 @@
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/canonical/k8s/cmd/k8s/errors"
+	"github.com/spf13/cobra"
+)
+
+func newLocalNodeStatusCommand() *cobra.Command {
+	localNodeStatusCmd := &cobra.Command{
+		Use:     "local-node-status",
+		Short:   "Retrieve the current status of the local node",
+		Hidden:  true,
+		PreRunE: chainPreRunHooks(hookSetupClient),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			defer errors.Transform(&err, nil)
+
+			clusterStatus, err := k8sdClient.NodeStatus(cmd.Context())
+			if err != nil {
+				return fmt.Errorf("failed to get cluster status: %w", err)
+			}
+
+			fmt.Println(clusterStatus)
+			return nil
+		},
+	}
+	return localNodeStatusCmd
+}
```

Comment:
Are we keeping this as a hidden command? What if we could repurpose it and make it a hidden subcommand of the k8s status? If it does not deliver relevant information, I'm OK to just dropping it.

Reply:
This is really just a hidden command for testing and debugging purpose. We could drop it but I also don't see a problem with keeping it.